api.planx.uk/server.ts

```diff
@@ -640,14 +640,25 @@ app.post("/webhooks/hasura/send-slack-notification", sendSlackNotification);
 
 app.use("/proxy/ordnance-survey", useOrdnanceSurveyProxy);
 
+app.get("/error", async (res, req, next) => {
+  try {
+    throw Error("This is a test error")
+  } catch (error) {
+    next(error)
+  }
+})
+
 const errorHandler: ErrorRequestHandler = (errorObject, _req, res, _next) => {
   const { status = 500, message = "Something went wrong" } = (() => {
-    if (errorObject.error) {
-      reportError(errorObject.error);
+    if (errorObject instanceof Error && airbrake) {
```

Comment:
If you look at #1332 there are actually some changes here I needed to make to hit this `if` block. `errorObject.error` is undefined, so this wasn't getting hit.

I'll also look at some tests for this tomorrow to make sure we're calling Airbrake when we think we are.

Reply:
There is some inconsistency with how we're currently handing errors I think - sometimes it's `next(error)` and other times we use `next({error, message: "something"})`.